frontend/dreamboard/src/app/services/components-communication.service.ts

```diff
@@ -79,4 +81,8 @@ export class ComponentsCommunicationService {
   userLoggedIn(loggedIn: boolean) {
     this.userLoggedInSource.next(loggedIn);
   }
+
+  updateScenes(scenes: VideoScene[]) {
```

Comment:
Hi Natalie, thanks for working on this!

QQ on this, why is this function needed? The scenes array is a reference shared across al scenes, when you update a specific scene, the array will be updated. I just want to confirm if this is a different case. Thank you!